tests/ui/job-view/fuzzy_test.jsx

```diff
@@ -0,0 +1,235 @@
+import Fuse from 'fuse.js';
+
+const testFuseArray = [
```

Comment:
This and the following put them as files under `tests/ui/mock`.
Look at other ui tests on how the import mock files.
As we spoke on Riot use real data.